From 1bcd26ed348d38a95d35bb6821be463e2be7150d Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:48:59 -0400
Subject: [tools] perf/timechart: Remove misplaced __maybe_unused

Message-id: <1408953210-26343-52-git-send-email-jolsa@redhat.com>
Patchwork-id: 88759
O-Subject: [PATCH RHEL7.1 BZ1133083 051/322] perf timechart: Remove misplaced __maybe_unused
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit 972ec653c489fd08ac2904caa98a6fa3f44dc303
Author: Arnaldo Carvalho de Melo <acme@redhat.com>
Date: Wed Nov 27 16:32:56 2013 -0300

description
===========
The 'event' parameter _is_ used.

echo Link: http://lkml.kernel.org/n/tip-`ranpwd -l 24`@git.kernel.org
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/builtin-timechart.c b/tools/perf/builtin-timechart.c
index c246f02..680632d 100644
--- a/tools/perf/builtin-timechart.c
+++ b/tools/perf/builtin-timechart.c
@@ -511,7 +511,7 @@ typedef int (*tracepoint_handler)(struct perf_evsel *evsel,
       const char *backtrace);
 
 static int process_sample_event(struct perf_tool *tool __maybe_unused,
-    union perf_event *event __maybe_unused,
+    union perf_event *event,
     struct perf_sample *sample,
     struct perf_evsel *evsel,
     struct machine *machine __maybe_unused)
-- 
1.7.1